src/cmd/query.rs

```diff
@@ -91,29 +91,29 @@ impl Query {
 
     fn get_fzf() -> Result<FzfChild> {
         let mut fzf = Fzf::new()?;
-        if let Some(fzf_opts) = config::fzf_opts() {
+        if let Some(mut fzf_opts) = config::fzf_opts() {
+            if let Some(fzf_extra_opts) = config::fzf_extra_opts() {
```

Comment:
This is the right way to use an `if let`, since there's no other execution path.